cfgov/research_reports/models.py

```diff
@@ -0,0 +1,235 @@
+import os
+
+from django.conf import settings
+from django.core import management
+from django.db import models
+from django.utils.safestring import mark_safe
+
+from wagtail.admin.edit_handlers import (
+    FieldPanel, InlinePanel, MultiFieldPanel, ObjectList, TabbedInterface
+)
+from wagtail.admin.forms.models import WagtailAdminModelFormMetaclass
+from wagtail.admin.forms.pages import WagtailAdminPageForm
+from wagtail.core.models import PageManager
+from wagtail.documents.edit_handlers import DocumentChooserPanel
+
+from modelcluster.fields import ParentalKey
+from modelcluster.models import ClusterableModel
+
+from v1.models import SublandingFilterablePage
+from v1.models.base import CFGOVPage
+
+
+alpha_map = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+
+
+def get_report_parts(is_appendix=False):
```

Comment:
I know you mention tests in your TODOs -- those would really help understand how this code works.